WebDriverAgentTests/UnitTests_tvOS/Doubles/XCUIElementDouble.h

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2018-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import <Foundation/Foundation.h>
+#import <WebDriverAgentLib/FBElement.h>
+#import <XCTest/XCUIElementTypes.h>
+
+@class XCUIApplication;
+
+@interface XCUIElementDouble : NSObject<FBElement>
+@property (nonatomic, strong, nonnull) XCUIApplication *application;
+@property (nonatomic, readwrite, assign) CGRect frame;
+@property (nonatomic, assign) BOOL fb_isObstructedByAlert;
+@property (nonatomic, readwrite, copy, nonnull) NSDictionary *wdRect;
+@property (nonatomic, readwrite, assign) CGRect wdFrame;
+@property (nonatomic, readwrite, copy, nonnull) NSString *wdUID;
+@property (nonatomic, copy, readwrite, nullable) NSString *wdName;
+@property (nonatomic, copy, readwrite, nullable) NSString *wdLabel;
+@property (nonatomic, copy, readwrite, nonnull) NSString *wdType;
+@property (nonatomic, strong, readwrite, nullable) NSString *wdValue;
+@property (nonatomic, readwrite, getter=isWDEnabled) BOOL wdEnabled;
+@property (nonatomic, readwrite, getter=isWDVisible) BOOL wdVisible;
+@property (nonatomic, readwrite, getter=isWDAccessible) BOOL wdAccessible;
+#if TARGET_OS_TV
```

Comment:
does it make sense to have such ifdef here? I assume there tests will be exclusively executed for this platform